mvp.feature

```diff
@@ -1102,3 +1104,39 @@ Feature: MVP features for NVA
       | Institution DNS               |
       | Administration ID             |
       | Feide Organization ID         |
+
+  @902
+  Scenario: User opens a Public Profile from a Public Page
+    Given the user has opened NVA
+    And sees a Public Page
+    When they click a contributor
+    Then the contributors public profile page is opened
+
+  @913
+  Scenario: User sees published publications
+    Given the user is logged in as Creator
+    And has opened My Publications page
```

Comment:
Disse to linjene kan også byttes ut med denne:
`Given Creator opens My Publications`